aidants_connect_web/views/usagers.py

```diff
@@ -75,6 +75,24 @@ def _get_usagers_dict_from_mandats(mandats: Mandat) -> dict:
     usagers_without_mandats = set()
     delta = settings.MANDAT_EXPIRED_SOON
     for mandat in mandats:
+        expired_over_a_year = (
+            mandat.expiration_date
+            if mandat.expiration_date < now() - timedelta(365)
+            else False
+        )
```

Comment:
```suggestion
        expired_over_a_year = (mandat.expiration_date < now() - timedelta(365))
```

(Pas sûre de la nécessité des parenthèses)

Reply:
les 2 bouts de code ne font pas la même chose. dans le code de la PR expired_over_a_year vaut soit mandat.expiration_date ( donc une date) soit False. 

dans le commit suggéré ça vaut soit True soit False. 

après j'ai l'impression que la valeur de la date qui est mise dans  expired_over_a_year soit très utile, donc mettre True c'est ok 

par contre les parenthèses sont pas nécessaire  effectivement.